WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -101,10 +101,10 @@ export default function Delegates() {
           onChange={(__, { checked }) => setToggleAdmin(checked)}
         />
       )}
-      <Grid container>
+      <Grid>
         <Grid.Column only="computer" computer={4}>
           <Header>{I18n.t('delegates_page.regions')}</Header>
-          <Menu vertical>
+          <Menu vertical fluid>
```

Comment:
What does this line do?

Reply:
Without that, the UI will be as follows. `fluid` makes it to use the available width completely. According to the documentation in semantic UI site, the definition of fluid is "A vertical menu may take the size of its container".

<img width="1512" alt="Screenshot 2024-02-03 at 21 12 30" src="https://github.com/thewca/worldcubeassociation.org/assets/9512698/474cc003-ed95-4129-923d-5cb18dc4d3e7">